test/base/LendingPoolRepayTest.js

```diff
@@ -41,20 +41,21 @@ contract('LendingPoolRepayTest', function (accounts) {
     });
 
     withData({
-        _1_cTokenSupported_basic: [accounts[1], loansAddress, true, true, 10, false, undefined, false],
-        _2_cTokenSupported_notLoan: [accounts[1], accounts[2], true, true, 10, false, 'ADDRESS_ISNT_LOANS_CONTRACT', true],
-        _3_cTokenSupported_transferFail: [accounts[1], loansAddress, true, false, 200, false, "LENDING_TRANSFER_FROM_FAILED", true],
-        _4_cTokenSupported_compoundFail: [accounts[1], loansAddress, true, true, 10, true, 'COMPOUND_DEPOSIT_ERROR', true],
-        _6_cTokenNotSupported_basic: [accounts[1], loansAddress, false, true, 10, false, undefined, false],
-        _7_cTokenNotSupported_notLoan: [accounts[1], accounts[2], false, true, 10, false, 'ADDRESS_ISNT_LOANS_CONTRACT', true],
-        _8_cTokenNotSupported_transferFail: [accounts[1], loansAddress, false, false, 200, false, "LENDING_TRANSFER_FROM_FAILED", true],
+        _1_cTokenSupported_basic: [accounts[1], loansAddress, true, true, 10, false, 1000, undefined, false],
+        _2_cTokenSupported_notLoan: [accounts[1], accounts[2], true, true, 10, false, 1000, 'ADDRESS_ISNT_LOANS_CONTRACT', true],
+        _3_cTokenSupported_transferFail: [accounts[1], loansAddress, true, false, 200, false, 1000, "LENDING_TRANSFER_FROM_FAILED", true],
+        _4_cTokenSupported_compoundFail: [accounts[1], loansAddress, true, true, 10, true, 1000, 'COMPOUND_DEPOSIT_ERROR', true],
+        _6_cTokenNotSupported_basic: [accounts[1], loansAddress, false, true, 10, false, 1000, undefined, false],
+        _7_cTokenNotSupported_notLoan: [accounts[1], accounts[2], false, true, 10, false, 1000, 'ADDRESS_ISNT_LOANS_CONTRACT', true],
+        _8_cTokenNotSupported_transferFail: [accounts[1], loansAddress, false, false, 200, false, 1000, "LENDING_TRANSFER_FROM_FAILED", true],
```

Comment:
Is it possible to add a new scenario like _6_ but the allowance 0 (or less than amountToRepay) to get an allowance error?

Reply:
Added